Gaijinizer patch/CommonEvents_5.txt

```diff
@@ -4,33 +4,34 @@
 
 ガマン、ガマン♪
 > CONTEXT: CommonEvents_5/7/Dialogue/enemy2
-Empuse Chan, you can't get tempted, huh?
-hold back, hold back♪#MTLed
+Empuse-chan, don't let yourself be tempted, okay?
+Patience, patience. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 
 ヘカーテの言葉で、エンプーサの表情は元に戻ってしまった…
 > CONTEXT: CommonEvents_5/13/Dialogue
 > CONTEXT: CommonEvents_180/29/Dialogue
-In the words of Hecate, the expression of Empuse has been
-restored ...#MTLed
+Empuse's expression has gone back to normal after
+hearing Hectate's words.#Ron
 > END STRING
 
 > BEGIN STRING
 おおっ！！\\N[1]くん！
 
 服を脱いでやる気満々だ！！
 > CONTEXT: CommonEvents_5/27/Dialogue/enemy6
-Ooh!\\N[1]-kun! I am unwilling to take off my
-clothes!!#MTLed
+Ooh!\\N[1]-kun!
+
+Aren't we eager to strip off!!#Ron
 > END STRING
 
 > BEGIN STRING
 それともヤられる気満々なのかな！？
 
 どちらにせよ楽しみです！！
 > CONTEXT: CommonEvents_5/31/Dialogue/enemy6
-Or maybe it's a temptation to do it!?
-I am looking forward to it anyway!!#MTLed
+Or is it maybe you're eager to be raped!?
+Either way I'm looking forward to it!!#Ron
 > END STRING
```

Comment:
Adding a word into here to make it flow a bit better:
>_**Or maybe it's just because you're eager to be raped!?
Either way, I'm looking forward to it!!**_